src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -352,6 +352,7 @@ private Movie GetMovie(ParsedMovieInfo parsedMovieInfo, string imdbId, SearchCri
             // TODO: Answer me this: Wouldn't it be smarter to start out looking for a movie if we have an ImDb Id?
             if (!String.IsNullOrWhiteSpace(imdbId) && imdbId != "0")
             {
+                _logger.Debug("Try matching by imdbID");
```

Comment:
@silv3r23 Nah, I would really like those lines removed, as we are already spamming the logs enough as it is when an RSS Sync is happening. Especially bad, when you are debugging and trying to work out the VS logs.